### PR TITLE
LPS-155085 - Only allow alphanumeric names with dashes and underlines for icon packs

### PR DESCRIPTION
**Pull request by bryceosterhaus**
https://issues.liferay.com/browse/LPS-155085

Made some changes for Brian and utilized Validator utils

### CODE REVIEW

**Comment by bryceosterhaus** [comment]
ci:stop

**Comment by bryceosterhaus** [comment]
ci:test:sf

**Comment by bryceosterhaus** [comment]
ci:test:relevant

File: modules/apps/frontend-icons/frontend-icons-web/src/main/resources/META-INF/resources/js/pages/instance_settings/AddIconPackModal.tsx [hunk 1]
```diff
@@ -68,7 +68,17 @@ export default function AddIconPackModal({
 		},
 	});
 
+	const nameTaken = !existingIconPackName && !!icons[iconPackName];
+
+	const containsWhiteSpace = !!iconPackName.match(/\s/);
```
Comment (kresimir-coko) [inline modules/apps/frontend-icons/frontend-icons-web/src/main/resources/META-INF/resources/js/pages/instance_settings/AddIconPackModal.tsx:73]: I think that the empty line between these 2 declarations is unnecessary, as we don't use `nameTaken` on line 73. But it's whatever 🤷

**Review by kresimir-coko — APPROVED** [review]
LGTM 👍

**Comment by susanchen3** [comment]
Reviewed. Test failures are unrelated :heavy_check_mark: Looks good from QA perspective. :) 

- Semantic versioning failure: not unique to this PR

**Pull request closed without merging**